server/validators/dateInFuture.test.ts

```diff
@@ -0,0 +1,29 @@
+import { formatISO } from 'date-fns'
+import dateInFuture from './dateInFuture'
+
+describe('dateInFuture', () => {
```

Comment:
same as above, could it make sense to have this in a separate repo and load it as a dependency? 
Same applied to the other classes in this directory